tests/nnstreamer_filter_python3/runTest.sh

```diff
@@ -37,13 +37,11 @@ if [[ -d $PATH_TO_PLUGIN ]]; then
     fi
 else
     echo "No build directory"
-    report
-    exit
 fi
 
 FRAMEWORK="python3"
 # This symlink is necessary only for testcases; when installed, symlinks will be made
-pushd ../../build/ext/nnstreamer/tensor_filter
+pushd ../../build/ext/nnstreamer/tensor_filter || echo "INFO: ../../build/ext/nnstreamer/tensor_filter folder not found" && pushd ${PWD}
```

Comment:
If a built dir exists, an error occurs because `../../build/ext/nnstreamer/tensor_filter` is pushed twice.
Check the result of the `pushd`.

Reply:
yes correct, I updated the code